### ISSUE TITLE
Bug? Multiple entries for the same story

### ISSUE THREAD

**Issue by trungdq88** [samuelclay/NewsBlur#1470]
Hi @samuelclay,

Not sure why this RSS is currently rendered as multiple stories in Newsblur: https://devutils.app/changelog.rss

Other RSS readers can parse the URL correctly but NewsBlur shows this:

![newsblur](https://user-images.githubusercontent.com/4214509/117096326-48550680-ad9b-11eb-8ef6-669655056850.png)

Thanks for taking a look.

**Comment by samuelclay** [comment]
It's because the feed doesn't validate. The feed is explicitly saying the guids are all the same, so NewsBlur helpfully teases them apart. But let the publisher know that their feed isn't validating.

https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fdevutils.app%2Fchangelog.rss

**Comment by samuelclay** [comment]
The feed xml has this inside as well:

```
<link>https://devutils.app/changelog/</link><guid isPermaLink="true">https://devutils.app/changelog/</guid><pubDate>Invalid Date</pubDate></item>
```

**Comment by samuelclay** [comment]
I agree that NewsBlur shouldn't be replicating this story over and over. The fact that it's this reproducible means I can get it fixed. I'll fix it today.

**Comment by samuelclay** [comment]
Fixed in c7fac210d. Doesn't work retroactively but they'll stop coming up as new, even with the broken date.
